adapters/appnexus/appnexus.go

```diff
@@ -471,17 +473,3 @@ func resolvePlatformID(platformID string) int {
 
 	return defaultPlatformID
 }
-
-func loadCategoryMapFromFileSystem() map[string]string {
-	// Load custom options for our adapter (currently just a lookup table to convert appnexus => iab categories)
-	opts, err := os.ReadFile("./static/adapter/appnexus/opts.json")
-	if err == nil {
-		var adapterOptions appnexusAdapterOptions
```

Comment:
Now that we have in memory map for IAB categories, can we also remove `appnexusAdapterOptions` struct?

Reply:
good catch.  https://github.com/prebid/prebid-server/pull/2619/commits/03459f1350f2644165a2f878c0aaddb9373899a7 removes `appnexusAdapterOptions`